packages/thirdweb/src/auth/verify-signature.ts

```diff
@@ -103,9 +113,35 @@
   accountFactory,
 }: VerifyContractWalletSignatureParams) {
   const messageHash = hashMessage(message);
+
+  const parsedSignature = (() => {
```

Comment:
this seems a bit scary? maybe add tests for these sub cases?

Reply:
Added a few tests and removed signature object support (we can't support it until we re-add 1271 support. What's scary about it though? It's just parsing the input to the format we prefer